.github/workflows/renovate.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# This workflow runs a "self-hosted" Renovate configuration. We're using this
+# instead of the Renovate GitHub App because the app can only be configured
+# at the GitHub organization level, and it's unwieldy for that configuration to
+# be shared across the giant microsoft org.
+#
+# For general documentation on running Self-Hosted Renovate using their GitHub
+# action like this, see:
+#
+#   * https://github.com/renovatebot/github-action
+#   * https://docs.renovatebot.com/self-hosted-configuration
+#
+# This workflow is currently an experiment to test Renovate on a small scale.
+# If we like how it works in this repo, we would probably want to create an
+# independent repo like microsoft/accessibility-insights-renovate-config to
+# contain this workflow + the self-host config + a shared Renovate preset,
+# and have the workflow run once against all our repos rather than repeating
+# the config and CI cost for each repo.
+
+name: Renovate
+
+# Note that the triggers for this workflow only control how often renovate
+# should check to see if there are updates to perform. Certain updates (eg,
+# rebasing open PRs when yarn.lock is updated in main) will happen immediately
+# on the next update check, but new PRs will only be opened according to the
+# schedule property of /.github/renovate.json5
+#
+# Note that this requires a GitHub PAT (secrets.RENOVATE_TOKEN) which has more
+# privileges than is safe to use with PRs from forks. *Do not* add triggers
+# against pull_request (or to "push" with non-protected branches).
+on:
+  push:
+    branches: ['main']
+    paths:
+      - '**/package.json'
+      - '**/yarn.lock'
+  schedule:
+    # This reads "every 15 minutes on weekdays from 7am-7pm PST (3pm-3am UTC),
+    # and hourly outside of that".
+    - cron: '15,30,45 15,16,17,18,19,20,21,22,23,0,1,2,3 * * MON-FRI'
```

Comment:
this is one of the more exciting cron schedules I've seen in a while.

Reply:
Yeah; `15-23,0-3` should be valid and would have been more concise, but the syntax checker in vs code didn't like it and I didn't think writing it out was so onerous that it was worth leaving the red squiggle there to confuse someone later, even if it ended up working in practice